clojure-mode.el

```diff
@@ -1943,6 +1946,65 @@ This will skip over sexps that don't represent objects, so that ^hints and
           (backward-sexp 1))
         (setq n (1- n))))))
 
+(defun clojure-top-level-comment-p ()
```

Comment:
I think we should make this more generic by making it take `comment` or whatever other form as a param.

Reply:
This would just return the first form of a sexp? or compare the first form to the arg passed in?